core/domain/base_model_validators.py

```diff
@@ -757,6 +757,8 @@ def _validate_commit_cmds_schema(cls, item):
                 continue
             try:
                 change_domain_object(commit_cmd_dict)
+            except utils.DeprecatedError:
+                pass
```

Comment:
Should we at least log a warning here?

Reply:
I don't think there is a need to log a warning, because we are already keeping a track of the deprecated commands and their values using the list.

Reply:
Right. Maybe just add a comment here, to explain what is going on.

Reply:
Done.